src/System.Private.CoreLib/shared/System/Threading/ThreadPool.cs

```diff
@@ -911,8 +911,8 @@ internal sealed class _ThreadPoolWaitOrTimerCallback
         private WaitOrTimerCallback _waitOrTimerCallback;
         private ExecutionContext? _executionContext;
         private object? _state;
-        private static readonly ContextCallback _ccbt = new ContextCallback(WaitOrTimerCallback_Context_t);
-        private static readonly ContextCallback _ccbf = new ContextCallback(WaitOrTimerCallback_Context_f);
+        private static readonly ContextCallback _ccbt = new ContextCallback(WaitOrTimerCallback_Context_t!); // TODO-NULLABLE: https://github.com/dotnet/roslyn/issues/35896
+        private static readonly ContextCallback _ccbf = new ContextCallback(WaitOrTimerCallback_Context_f!); // TODO-NULLABLE: https://github.com/dotnet/roslyn/issues/35896
```

Comment:
Similarly, the `object state` for all of these should be `object? state` and the TODOs and the !s here should be removed.

Reply:
Done